example/vacanza.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Ruff (0.15.9)

[warning] UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)